src/lib.rs

```diff
@@ -27,10 +27,11 @@ use std::{
 
 use xenctrl_sys::{
     xc_dominfo_t, xc_error_code_XC_ERROR_NONE, xc_error_code_XC_INTERNAL_ERROR, xc_interface,
-    xenmem_access_t, xenmem_access_t_XENMEM_access_n, xenmem_access_t_XENMEM_access_r,
-    xenmem_access_t_XENMEM_access_rw, xenmem_access_t_XENMEM_access_rwx,
-    xenmem_access_t_XENMEM_access_rx, xenmem_access_t_XENMEM_access_w,
-    xenmem_access_t_XENMEM_access_wx, xenmem_access_t_XENMEM_access_x, xentoollog_logger,
+    xc_vcpuinfo_t, xenmem_access_t, xenmem_access_t_XENMEM_access_n,
```

Comment:
when using types from the sys crate for the signature of public function, they should be added to the list of re-exported types instead